pom.xml

```diff
@@ -26,7 +26,21 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.9.1</version>
```

Comment:
[Use 4 version](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#dont-use-any-other-version-of-junit) Not sure if this is critical, but since it's in common mistakes i'll mention it.

Reply:
i can't find 4 version for junit-jupiter-api, 
mb this is not critical)

Reply:
<dependency>
            <groupId>junit</groupId>
            <artifactId>junit</artifactId>
            <version>4.12</version>
            <scope>test</scope>
</dependency> 
you had this dependency in your task. You don’t need to add any more dependency